gems/pending/MiqVm/MiqVm.rb

```diff
@@ -64,7 +70,7 @@ def vmDisks
   end
 
   def openDisks(diskFiles)
-    pVolumes = []
+    pVolumes = Array.new
```

Comment:
Because RuboCop said to change it.  It likes [] instead of Array.new and {} instead of Hash.new.